go.mod

```diff
@@ -1,22 +1,16 @@
-module github.com/crossplaneio/minimal-gcp
+module github.com/crossplaneio/stack-minimal-gcp
 
 go 1.13
 
+replace github.com/crossplaneio/resourcepacks => github.com/muvaf/resourcepacks v0.0.0-20200124113357-9b90b932f381
```

Comment:
go ahead and remove this replace statement now, and update this go.mod to use the latest `github.com/crossplaneio/resourcepacks`